test/unittests/parser_test.cpp

```diff
@@ -95,12 +95,33 @@ TEST(parser, limits_invalid)
     EXPECT_THROW(parse_limits(input.data()), parser_error);
 }
 
-TEST(parser, locals)
+TEST(parser, code_locals)
 {
-    const auto input = "81017f"_bytes;
-    const auto [l, p] = parse<Locals>(input.data());
-    EXPECT_EQ(l.count, 0x81);
-    EXPECT_EQ(l.type, ValType::i32);
+    const auto wasm_locals = "81017f"_bytes;  // 0x81 x i32.
+    const auto wasm =
+        bytes{wasm_prefix} + make_section(uint8_t(SectionId::code),
```

Comment:
I would rather keep using static section numbers and not exposing `types.h`

Reply:
Ok, no problem.